tests/extractors/test_bigquery_extractor.py

```diff
@@ -67,11 +74,12 @@ def test_extract(self):
 
         assert 'customers' == dataset.name
 
-    @mock.patch("airflow.models.TaskInstance")
-    @mock.patch("google.cloud.bigquery.Client")
-    @mock.patch("json.dumps")
-    def test_extract_on_complete(self, mock_json, mock_client, mock_ti):
-        log.info("test_extract_on_complete")
+    @mock.patch("jsonpickle.encode")
```

Comment:
why do you need to mock this one? This doesn't seem to be part of the contract.

Reply:
done.